### PR TITLE
Misc. changes (mostly comments)

### PR DESCRIPTION
**Pull request by CalabashSquash**
misc fixed

### CODE REVIEW

**Review by chrdevmar — APPROVED** [review]
lgtm

File: contracts/implementation/KeeperRewards.sol [hunk 1]
```diff
@@ -52,6 +52,7 @@ contract KeeperRewards is IKeeperRewards {
         try IOracleWrapper(ILeveragedPool(_pool).settlementEthOracle()).poll() {} catch Error(string memory reason) {
             emit PoolUpkeepError(_pool, reason);
         }
+        // `IOracleWrapper` is typically implemented such that `getPrice` returns a number with 18 decimal places.
```
Comment (dospore) [inline contracts/implementation/KeeperRewards.sol:55]: As a non sc dev me this comment doesnt provide much value / information to me

**Review by iflp — APPROVED** [review]
lgtm

File: contracts/implementation/PoolCommitter.sol [hunk 2]
```diff
@@ -894,7 +895,7 @@ contract PoolCommitter is IPoolCommitter, IPausable, Initializable {
      */
     function setMintingFee(uint256 _mintingFee) external override onlyFeeController {
         mintingFee = PoolSwapLibrary.convertUIntToDecimal(_mintingFee);
-        require(mintingFee < MAX_MINTING_FEE, "Minting fee >= 100%");
+        require(mintingFee <= MAX_MINTING_FEE, "Minting fee > 100%");
```
Comment (dospore) [inline contracts/implementation/PoolCommitter.sol:898]: 100% minting fee would sting

**Pull request merged**